Homework-2-In-Memory-CRUD-REST-Service/src/helpers/usersHelper.js

```diff
@@ -0,0 +1,12 @@
+export const shuffleUsers = array => {
```

Comment:
Oh man. First of all, why would you want different results for the same request to the server. Another thing: mutating argument and not returning anything? Is that even legal?

Reply:
Auto-suggest is just a method to filter items by substring. Imagine search field which shows possible options while user types some text.
Btw `sort` also mutates array.